### PR TITLE
adjustable cambrinth charges

### PR DESCRIPTION
**Pull request by Rhayle**
adds a setting to base.yaml and common-arcana to allow auto_mana to use variable number of cambrinth charges. Defaults to 4

### CODE REVIEW

File: common-arcana.lic [hunk 1]
```diff
@@ -458,7 +458,8 @@ module DRCA
       }]
     end
     total_cambrinth_cap = settings.cambrinth_items.map { |x| x['cap'] }.inject(&:+)
-    charges_count_floor = 4
+    charges_count_floor = settings.cambrinth_num_charges if (remaining >= settings.cambrinth_num_charges)
```
Comment (jandersson) [inline common-arcana.lic]: I think you might be able to combine the two conditional assignments
```ruby
charges_count_floor = if remaining >= settings.cambrinth_num_charges
                                       settings.cambrinth_num_charges
                                    else
                                       1
                                    end
```
or possibly a ternary 
```ruby
charges_count_floor = if remaining >= settings.cambrinth_num_charges ? settings.cambrinth_num_charges : 1
```

**Comment by jandersson** [comment]
How does this work with multiple cambrinth items?

**Comment by rpherbig** [comment]
@Rhayle how was this tested?

**Comment by Rhayle** [comment]
I run everything on absidian before creating a pr

**Pull request merged**